### ISSUE TITLE
"Close" button in most modals missing keyboard focus and accessible name

### ISSUE THREAD

**Issue by kingalg** [TryQuiet/quiet#1592]
**Priority** - high, users will not know how to close modal and this component is used in multiple places in the app.

**Issue** - no description for the button, for VO it only says “button”, no keyboard focus - keyboard users can't 
**Solution** - Add aria-label="Close", add tabindex="0"

<img width="617" alt="Screenshot 2023-06-27 at 13 04 58" src="https://github.com/TryQuiet/quiet/assets/68909168/50e1a938-e980-4421-9d03-df155efbc566">
